src/utils/arappUtils.ts

```diff
@@ -25,28 +25,34 @@ export function readArappIfExists(): AragonAppJson | undefined {
 
 /**
  * Returns app ens name.
+ * @param network "mainnet"
  * @return "voting.open.aragonpm.eth"
  */
-export function getAppEnsName(): string {
+export function getAppEnsName(network = 'default'): string {
   const arapp = readArapp()
 
-  const defaultEnvironment = arapp.environments.default
-  if (!defaultEnvironment) {
-    throw new BuidlerPluginError('Default environemnt not found in arapp.json')
+  const environment: AragonEnvironment | undefined =
+    arapp.environments[network] || arapp.environments['default']
```

Comment:
We probably don't need the fallback `arapp.environments['default']` since we're already defaulting `network`

Reply:
I think in current logic is necessary in case the network doesn't have an environment we try to use the default config. But we discuss with Lion that the `appName` was a bit clunky and he is working to improve the parsing.

Reply:
Code removed in #120